OracleTuxedo/core/dockerfiles/buildDockerImage.sh

```diff
@@ -0,0 +1,161 @@
+
+EOF
+exit 0
+}
+
+# Validate packages
+checksumPackages() {
+  echo "Checking if required packages are present and valid..."
+  if [ -z "$MD5VALUE" ]; then   # -m option is a MUST
+    echo "You must specify "$INSTALLER" MD5 value with -m option"
+    exit
+  fi
+  MD5="${MD5VALUE}  ${INSTALLER}"
+  MD5_CHECK="`md5sum ${INSTALLER}`"
+
+  if [ "$MD5" != "$MD5_CHECK" ]
+  then
+    echo "MD5 does not match! Download again!"
+    exit
+  fi
+
+#  md5sum -c Checksum.$DISTRIBUTION
+#  if [ "$?" -ne 0 ]; then
+#    echo "MD5 for required packages to build this image did not match!"
+#    echo "Make sure to download missing files in folder $VERSION. See *.download files for more information"
+#    exit $?
+#  fi
+}
+
+if [ "$#" -eq 0 ]; then usage; fi
+echo "====================="
+#
+
+VERSION=
+INSTALLER=
+MD5VALUE=
+SKIPMD5=0
+while getopts "hsi:m:v:" optname; do
+  case "$optname" in
+    "h")
+      usage
+      ;;
+    "s")
+      SKIPMD5=1
+      ;;
+    "i")
+     INSTALLER="$OPTARG"
+      ;;
+    "m")
+     MD5VALUE="$OPTARG"
+      ;;
+    "v")
+      VERSION="$OPTARG"
+      ;;
+    *)
+    # Should not occur
+      echo "Unknown error while processing options inside buildDockerImage.sh"
+      ;;
+  esac
+done
+
+if [ -z "$VERSION" ]; then   # -v option is a MUST
+    echo "You must specify Tuxedo version with -v option"
+    exit
+fi
+
+if [ -z "$INSTALLER" ]; then   # -i option is a MUST
+    echo "You must specify installer name with -i option"
+    exit
+fi
+
+
+if [ ! "$SKIPMD5" -eq 1 ]; then
+  checksumPackages
+else
+  echo "Skipped MD5 checksum."
+fi
+
+echo "====================="
+
+if [ ! -e ${INSTALLER} ]
+then
+  echo "Download the Tuxedo ZIP Distribution and"
+  echo "drop the file ${INSTALLER} in this folder before"
+  echo "building this Tuxedo Docker container!"
+  exit 
+fi
+
+# You may need uncomment the following lines and RP file name if you need check the RP md5sum value:
+#if [ ! -e p*_121300_Linux-x86-64.zip ]
+#then
+#  echo "Installing Tuxedo without any patches"
+#fi
+
+#MD5="3b311c87e921fa9df696bf74c39c3348  p19927652_121300_Linux-x86-64.zip"
+#MD5_CHECK="`md5sum p19927652_121300_Linux-x86-64.zip`"
+#
+#if [ "$MD5" != "$MD5_CHECK" ]
+#then
+#  echo "MD5 does not match! Download again!"
+#  exit
+#fi
+
+# Proxy settings
+PROXY_SETTINGS=""
+if [ "${http_proxy}" != "" ]; then
+  PROXY_SETTINGS="$PROXY_SETTINGS --build-arg http_proxy=${http_proxy}"
+fi
+
+if [ "${https_proxy}" != "" ]; then
+  PROXY_SETTINGS="$PROXY_SETTINGS --build-arg https_proxy=${https_proxy}"
+fi
+
+if [ "${ftp_proxy}" != "" ]; then
+  PROXY_SETTINGS="$PROXY_SETTINGS --build-arg ftp_proxy=${ftp_proxy}"
+fi
+
+if [ "${no_proxy}" != "" ]; then
+  PROXY_SETTINGS="$PROXY_SETTINGS --build-arg no_proxy=${no_proxy}"
+fi
+
+if [ "$PROXY_SETTINGS" != "" ]; then
+  echo "Proxy settings were found and will be used during build."
+fi
+
+echo "====================="
+
+# Fix up the locations of things
+cp ${VERSION}/tuxedo${VERSION}.rsp .
```

Comment:
Why are you copying files instead of just running `docker build` and specifying `${VERSION}/` as the build source?